Cargo.toml

```diff
@@ -7,7 +7,7 @@ license = "MIT"
 name = "cernan"
 readme = "README.md"
 repository = "https://github.com/postmates/cernan"
-version = "0.6.1"
+version = "0.6.2-pre"
```

Comment:
I've been marking new PRs with `-pre` if we've just come off a stable release. This just signals that it's not a full release. In the past we had 'stable' releases by version but with a distinct SHA.